lib/cc/yaml/nodes/engine_list.rb

```diff
@@ -24,7 +26,11 @@ def visit_unexpected(_visitor, _value, message = nil)
         private
 
         def verify_not_empty
-          error(EMPTY_ERROR_MESSAGE) if mapping.keys.empty? && errors.empty?
+          error(EMPTY_ERROR_MESSAGE) if errors.empty? && mapping.keys.empty?
+        end
+
+        def verify_enabled
+          error(NO_ENABLED_ERROR_MESSAGE) if errors.empty? && !mapping.values.map(&:enabled).include?(true)
```

Comment:
How about replacing `!map(&:enabled).include?(true)` with `!any?(&:enabled?)` (I think there are predicate aliases defined, but I'm not sure).